doc/Type/Attribute.pod6

```diff
@@ -124,47 +156,72 @@ default accessor for the attribute will return a writable value.
 
 =head2 X<trait is built|trait,is built (Attribute)>
 
-Defined as:
-
     multi sub trait_mod:<is>(Attribute:D $a, :$built!)
 
-By default, this trait allows setting up a I«private attribute» during object
-construction via C«.new». The same trait can be used to prevent setting up a
-I«public attribute» via C«.new» by passing it the boolean value C«False».
+By default, this trait allows setting up a I«private attribute» during
+object construction via C«.new». The same trait can be used to prevent
+setting up a I«public attribute» via C«.new» by passing it the boolean
+value C«False». Setting up an attribute with its value is ordinarily
+handled by assigning the value, but if C<:bind> is passed, then it will
+be bound instead:
 
     class Foo {
         has $!bar is built; # same as `is built(True)`
         has $.baz is built(False);
+        has $!qux is built(:bind);
 
-        method bar {
-            $!bar
-        }
+        method bar(::?CLASS:D:) { $!bar }
+        method qux(::?CLASS:D:) { $!qux }
     }
 
-    my $foo = Foo.new(bar => 1, baz => 2);
-    say $foo.bar; # «1␤»
-    say $foo.baz; # «Any␤»
+    my Foo:D $foo .= new: :bar[], :baz[], :qux[];
+    say $foo.bar.raku; # OUTPUT: «$[]␤»
+    say $foo.baz.raku; # OUTPUT: «Any␤»
+    say $foo.qux.raku; # OUTPUT: «[]␤»
 
 =head1 Methods
 
-The usual way to obtain an object of type C<Attribute> is by introspection:
-
-    class Useless {
-        has @!things;
-    }
-    my $a = Useless.^attributes(:local)[0];
-    say $a.raku;            # OUTPUT: «Attribute.new␤»
-    say $a.name;            # OUTPUT: «@!things␤»
-    say $a.package;         # OUTPUT: «(Useless)␤»
-    say $a.has_accessor;    # OUTPUT: «False␤»
-
-Modifying a private attribute from the outside is usually not possible, but
-since Attribute is at the level of the metaclass, all is fair game.
+=head2 method new
+
+=for code :skip-test<too long a method signature for one line>
```

Comment:
In principle, there should be no problem with so many lines, as long as you use =for code or =begin code